ts/input/mathml/MathMLCompile.ts

```diff
@@ -317,16 +350,24 @@ export class MathMLCompile<N, T, D> {
   }
 
   /**
-   * @param {string} text  The text to have leading/trailing spaced removed
+   * @param {string} text  The text to have spacing normalized
    * @return {string}      The trimmed text
    */
-  protected trimSpace(text: string): string {
+  protected normalizeSpace(text: string): string {
     return text.replace(/[\t\n\r]/g, ' ')    // whitespace to spaces
-               .replace(/^ +/, '')           // initial whitespace
-               .replace(/ +$/, '')           // trailing whitespace
                .replace(/  +/g, ' ');        // internal multiple whitespace
   }
 
+  /**
+   * @param {MmlNode} text  The token node whose leacing/trailing spaces should be removed
+   */
+  protected trimSpace(mml: MmlNode) {
+    let child = mml.childNodes[0] as TextNode;
```

Comment:
I thought there can now be multiple child nodes in a token. So why does that still only work on the first `textNode`?

Reply:
The trimming is of the initial spaces (which only occur on the first child) and the trailing spaces (which only occur on the last child, which is handled in lines 368 and 369 below.  So this is exactly what needs to be done to handle multiple child nodes.

Reply:
Sorry had not seen the answer on the front page: Sounds good. Thanks.